src/pages/nft/class/_classId/index.vue

```diff
@@ -456,9 +456,7 @@ export default {
       return this.NFTPrice !== undefined && this.NFTPrice > 0;
     },
     creatorMessage() {
-      return this.messageList.find(
-        element => element.event === 'mint_nft' || element.event === 'purchase'
-      );
```

Comment:
should we keep this as a fallback?

Reply:
I think it's not necessary. The `messageList` is mainly for showing messages from collectors. Messages from creators should be obtained using `NFTMetadata`.